ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -116,6 +116,7 @@ public int compare(IndexEntry e1, IndexEntry e2) {
   /**
    * Creates a new persistent index
    * @param datadir The directory to use to store the index files
+   * @param storeId The unique ID for the store that this this index represents.
```

Comment:
minor: remove repeated "this"

Reply:
also, does unique is required here. why not "Id of the store that this index represents"

Reply:
Changed